app/forms/upload_form.rb

```diff
@@ -41,10 +41,14 @@ def save_search_results(results)
 
   def scrape_search_results(results)
     results.map(&:id).each do |id|
-      SearchKeywordJob.perform_later(id)
+      SearchKeywordJob.set(wait: generate_delay_in_seconds).perform_later(id)
```

Comment:
🤔 With this, jobs can be in the queue simultaneously, as `generate_delay_in_seconds` return a random second, and the random number can be roundly equal, right?
I think a delay between jobs could be better. But let's go with this as we want to find a way to achieve it 💪

Reply:
@hoangmirs Indeed. The generated number can be even the same values. With current success rate, it's acceptable so I think we could go with the current implementation 🚀